stages/org.osbuild.grub2.iso

```diff
@@ -110,6 +123,8 @@ def main(root, options):
     vendor = options["vendor"]
     kdir = options["kernel"].get("dir", "/images/pxeboot")
     kopts = options["kernel"].get("opts")
+    cfg = options.get("config", {})
+    timeout = cfg.get("timeout", 0)
```

Comment:
You change the default from 60 to 0, is that intentional? It breaks backwards compatibility, and I am unsure from the commit-message whether that is intended. I think it is ok, just wondering what the exact reasoning is.

Reply:
yeah, 60 was just a bug to me and the only user of this is the simplified installer in osbuild-composer iirc so this shouldn't be an issue - we're setting a sane 5s there from now one which should be plenty

Reply:
True, although it is a bit iffy to change the default, I wonder if we should even have one? Maybe we should make it a requirement so we transfer that decision fully over to Composer?

Reply:
Having said this, it would be a breaking change, which is also not great. Maybe it is ok to change the default after all.

Reply:
It is going to be used from the image installer in Fedora, btw (see https://github.com/osbuild/osbuild-composer/pull/3077) :/

Reply:
> It is going to be used from the image installer in Fedora, btw (see https://github.com/osbuild/osbuild-composer/pull/3077) :/

is 60s correct there tho? I think it's merely a bug to fix but anyway, perhaps I can just leave 60s as default to avoid "breaking" anybody and then just set 5s in simplified-installer

Reply:
It comes from here: https://github.com/weldr/lorax/blob/79ca526fb97e4999baccfcc912ae00f253de9eef/share/templates.d/99-generic/config_files/x86/grub2-efi.cfg#L17

i.e. the standard anaconda templates.

Reply:
> i.e. the standard anaconda templates.

mistery solved thanks! back to what we need, is a 60s default ok then?

Reply:
I am afraid so, leave it at 60, but we should make it explicit in Composer _everywhere_.

Reply:
done, and explicitely setting it to 5s for simplified-installer here https://github.com/osbuild/osbuild-composer/pull/3131